encode/av1encode.c

```diff
@@ -0,0 +1,2964 @@
+#define LIBVA_UTILS_UPLOAD_DOWNLOAD_YUV_SURFACE 1
+
+#include <stdio.h>
+#include <stdbool.h>
+#include <string.h>
+#include <stdlib.h>
+#include <getopt.h>
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <sys/time.h>
+#include <sys/mman.h>
+#include <fcntl.h>
+#include <CHECK_CONDITION.h>
+#include <pthread.h>
+#include <errno.h>
+#include <math.h>
+#include <va/va.h>
+#include <va/va_enc_av1.h>
```

Comment:
redundant?

Reply:
Av1 has some specific structures defined in va_enc_av1.h